packages/front-end/components/Experiment/TabbedPage/RunningExperimentDecisionBanner.tsx

```diff
@@ -0,0 +1,152 @@
+import { Text, Flex, Box } from "@radix-ui/themes";
+import {
+  DecisionCriteriaData,
+  ExperimentInterfaceStringDates,
+  ExperimentResultStatusData,
+  VariationWithIndex,
+} from "back-end/types/experiment";
+import { useState } from "react";
+import { BsLightningFill } from "react-icons/bs";
+import Collapsible from "react-collapsible";
+import { FaAngleRight } from "react-icons/fa";
+import DecisionCriteriaModal from "@/components/DecisionCriteria/DecisionCriteriaModal";
+import Link from "@/components/Radix/Link";
+import ExperimentDecisionExplanation from "./ExperimentDecisionExplanation";
+
+interface Props {
+  experiment: ExperimentInterfaceStringDates;
+  runningExperimentStatus: ExperimentResultStatusData;
+  decisionCriteria: DecisionCriteriaData;
+  showDecisionCriteriaLink?: boolean;
+}
+
+export default function RunningExperimentDecisionBanner({
+  experiment,
+  runningExperimentStatus,
+  decisionCriteria,
+  showDecisionCriteriaLink = true,
+}: Props) {
+  const [showDecisionCriteria, setShowDecisionCriteria] = useState(false);
+
+  const variations = experiment.variations;
+  const indexedVariations = variations.map<VariationWithIndex>((v, i) => ({
+    ...v,
+    index: i,
+  }));
+
+  if (!runningExperimentStatus) return null;
```

Comment:
question: Do we need this? I don't think this can ever happen right (based on types)?